resource/tasks.json

```diff
@@ -15469,6 +15472,38 @@
             477
         ]
     },
+    "Sami@Roguelike@CheckCollapsalParadigms": {
+        "algorithm": "OcrDetect",
+        "action": "DoNothing",
+        "text": [
+            "去量化",
+            "去深量化",
+            "实质性坍缩",
+            "蔓延性坍缩",
+            "非线性移动",
+            "非线性行动",
+            "情绪实体",
+            "恐怖实体",
+            "泛社会悖论",
+            "泛文明悖论",
+            "气压异常",
+            "气压失序",
+            "触发性损伤",
+            "触发性危殆",
+            "趋同性消耗",
+            "趋同性缺失",
+            "目空一些",
+            "睁眼瞎",
+            "图像损坏",
+            "一抹黑"
+        ],
```

Comment:
We could use the [ResourceUpdater](https://github.com/MaaAssistantArknights/MaaAssistantArknights/blob/15cde5284b1f7685b209b383bf3dbae7dfd65761/tools/ResourceUpdater/main.cpp#L1134) for this (to ease updates and global adaptation).
I've already checked the gamedata and it's inside of "modules" -> "chaos"
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/6c28239a-d554-4cc9-a911-6a3a896f0cba) 
Currently that section of the ResourceUpdater is only used for the overseas clients to add the translated items automatically (the official json is not automatically populated yet)

Reply:
That sounds a good idea.
Oh and if chaos is its official name, I may adapt the class/variable names to make more sense.

Reply:
The global English name is "collapse paradigm", but "chaos" is the one in all documents and filrs

Reply:
Ok I do have used the correct name. Thanks for letting me know!